Sources/Datadog/Tracing/AutoInstrumentation/TracingHeaderType.swift

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// The type of the telemetry tracing header injected to requests.
+///
+/// - `dd` - Datadog's [Open Tracing header](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces).
+/// - `b3s` - Open Telemetry B3 [Single header](https://github.com/openzipkin/b3-propagation#single-headers).
+/// - `b3m` - Open Telemetru B3 [Multiple header](https://github.com/openzipkin/b3-propagation#multiple-headers).
```

Comment:
1/ The word "telemetry" looks odd in this context
2/ `x-datadog-*` are not "Open Tracing" headers - these are Datadog headers 🙂. The thing is that Open Tracing was deprecated (in favour of OTel) [before it managed to define its headers standard](https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md#trace-context-http-headers).

How about following changes?

```suggestion
/// The type of the tracing header injected to requests.
///
/// - `dd` - [Datadog's `x-datadog-*` header](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces).
/// - `b3s` - Open Telemetry B3 [Single header](https://github.com/openzipkin/b3-propagation#single-headers).
/// - `b3m` - Open Telemetry B3 [Multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers).
```

Reply:
I guess the user will be looking for keywords such as 'trace propagation' or 'distributed tracing'. Maybe we could use 'The type of trace propagation header injected to requests.' or the 'The trace propagation format injected to requests'.